serial_protocol.md

```diff
@@ -0,0 +1,72 @@
+#Protocol:
+During the first use, or when the chiplet changes environments a “NETWORK” call is expected to initialize the wifi parameters.
+
+Every time a serial connection is established we will expect a “INIT” call after which any subsequent calls can be made, until the connection is closed. 
+
+In the following examples we use $ to represent variables. For example $Host represents your firebase host.
+
+##Response Format Byte
+All responses will be prefixed with one of the following bytes signifying the response type.
+```
+  + If response is ok and a raw value
+  * If response is ok and a raw value prefixed by count of bytes in response. 
```

Comment:
That is actually where I saw the most use for it. Say you wanted to store a bunch of text to scroll on a display with line breaks. Or say you wanted to push a large binary blob, like a OTA update, to your client. This is where it is nice to know the size of the package so you can prepare memory and storage for it before you start reading it.